webots/protos/Zumo32U4.proto

```diff
@@ -3,9 +3,10 @@
 EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/appearances/protos/Rubber.proto"
 
 PROTO Zumo32U4 [
-  field SFVec3f    translation  0 0 0
-  field SFRotation rotation     0 0 1 0
-  field SFString   name         "robot"
+  field SFVec3f     translation     0 0 0
+  field SFRotation  rotation        0 0 1 0
+  field SFString    name            "robot"
+  field SFString    contactMaterial "default" # Contact material of the tracks
```

Comment:
Why make this variable? We could set it fixed to rubber.

Reply:
If we use the robot on other worlds, the standard contact properties are set to "default". The idea was to keep that and always specifiy in the instance of the proto which material we would like to use.